src/mjolnir/adminbuilder.cc

```diff
@@ -273,10 +271,8 @@ void BuildAdminFromPBF(const boost::property_tree::ptree& pt,
   }
 
   // loading SpatiaLite as an extension
-  if (!valhalla::mjolnir::load_spatialite(db_handle)) {
-    sqlite3_close(db_handle);
-    return;
-  }
+  auto* db_cache = spatialite_alloc_connection();
+  spatialite_init_ex(db_handle, db_cache, 0);
```

Comment:
this is basically the new syntax. `spatialite_init(0)` is deprecated. we need to hold a pointer to the connection's address